src/slashcommands/ServerColor.ts

```diff
@@ -0,0 +1,50 @@
+import { CommandInteraction, CacheType, MessageEmbed, ColorResolvable } from "discord.js";
+import { ApplicationCommandOptionTypes } from "discord.js/typings/enums";
+import Enmap from "enmap";
+import { Bot } from "../Bot";
+import { Option, Subcommand } from "./Option";
+import { SlashCommand } from "./SlashCommand";
+
+export const serverColors = new Enmap('serverColors');
+
+export class ServerColor implements SlashCommand {
+    name: string = 'servercolor';
+    description: string = '[MANAGER] Set the servers color to be used in other Arrays';
```

Comment:
Other arrays? Non coder users will have no idea what this means. I'm not even exactly sure what you mean by other arrays

Reply:
I have no idea what I meant by this